sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -68,27 +68,22 @@ function getTopicFilter(value: any): SqlRuleFilter | CorrelationRuleFilter {
 /**
  * @internal
  * @ignore
- * Helper utility to retrieve rule `action` value from given input,
- * or undefined if not passed in.
+ * Helper utility to retrieve rule `action` value from given input.
  * @param value
  */
-function getRuleActionOrUndefined(value: any): SqlRuleAction | undefined {
-  if (value == undefined) {
-    return undefined;
```

Comment:
How are we confident that we will get a non null/undefined value here?
If the intention of the change is to ensure we dont return undefined from here, I would recommend adding
```
if (value == undefined) { value = {} }
```

Reply:
Or we should allow sqlAction to be undefined when getting the result from the service.
name and filter are required for creating a rule, not action

Reply:
No, observed that this `val` is never undefined when tested, 
If the user sets action to be `undefined`, this `val` would be `{ '$': { 'i:type': 'EmptyRuleAction' } }` 
and we return the action to be...
```ts
{
  sqlExpression: undefined,
  sqlParameters: undefined,
  compatibilityLevel: undefined,
  requiresPreprocessing: undefined
}
```